extensions/sql-database-projects/src/tools/newProjectTool.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as nls from 'vscode-nls';
+import * as path from 'path';
+import * as constants from '../common/constants';
+
+const localize = nls.loadMessageBundle();
+const MaxDefaultProjectNameCounter: number = 99;
```

Comment:
why is the max 99?

Reply:
oh great point... Any recommendations on how to choose a max? I thought it'd be good _some_ max to prevent infinite looping